tree/ntuple/src/RColumnElement.cxx

```diff
@@ -61,7 +61,7 @@ std::pair<std::uint16_t, std::uint16_t> ROOT::Internal::RColumnElementBase::GetV
    default:
       if (type == kTestFutureColumnType)
          return std::make_pair(32, 32);
-      assert(false);
```

Comment:
I would not make these into `R__ASSERT`s: a crash here would mean that the developers (us) forgot to add this `case`, which is supposed to be "impossible" (it would be found during development / unit testing very quickly) and for such things I think an `assert` that disappears in release is a better fit.

Reply:
Yes, but if this "impossible" condition is somehow met in production, we want to know instead of silently ignoring it. The column type also (indirectly) comes from user input, so I think being extra careful is useful. If the "impossible" never happens, then there should be no difference between the two.